ArduPlane/quadplane.cpp

```diff
@@ -1410,7 +1410,12 @@ void SLT_Transition::update()
         (quadplane.transition_failure > 0) &&
         ((now - transition_start_ms) > ((uint32_t)quadplane.transition_failure * 1000))) {
         gcs().send_text(MAV_SEVERITY_CRITICAL, "Transition failed, exceeded time limit");
-        plane.set_mode(plane.mode_qland, ModeReason::VTOL_FAILED_TRANSITION);
+        if (quadplane.options & QuadPlane::OPTION_TRANS_FAIL_TO_FW) {
```

Comment:
check for ground speed is above half the transition airspeed threshold

Reply:
done